src/NetDaemon/apps/UnifiApp/DeviceTrackerApp.cs

```diff
@@ -33,11 +33,12 @@ public async Task InitializeAsync(CancellationToken cancellationToken)
 
         data.ClientDevices.SubscribeAsync(async data =>
         {
+            var currentTime = DateTime.UtcNow;
```

Comment:
Move variable declaration closer to usage.
By that I am saying that we don't need the currentTime variable at all in this class, it should be initialized in the DeviceTracker class and used there.

Update this class and change signature for DeviceTracker.SetState